src/build.ts

```diff
@@ -205,12 +206,17 @@ async function emit(bundle: RollupBuild, conf: any, makePure = false) {
   await fs.promises.mkdir(dir, {recursive: true}).catch(() => null)
   for (let file of result.output) {
     let content = (file as any).code || (file as any).source
-    if (makePure) content = addPureComments(content)
+    if (makePure) content = addPureComments(content);
+    let sourceMap:SourceMap = (file as any).map;
+    if (sourceMap) {
+      content = content + `\n//# sourceMappingURL=${file.fileName}.map`;
+      await fs.promises.writeFile(join(dir, file.fileName + ".map"), sourceMap.toString());
+    }
     await fs.promises.writeFile(join(dir, file.fileName), content)
   }
 }
 
-async function bundle(pkg: Package, compiled: Output) {
+async function bundle(pkg: Package, compiled: Output, generateSourceMap = false) {
```

Comment:
Could you make the last argument an options object, with `generateSourceMap` being the only option right now? That way the function signature doesn't become a mess if we ever add more options.